src/main/java/com/gluonhq/substrate/target/MacOSTargetConfiguration.java

```diff
@@ -55,6 +55,12 @@
     private static final List<String> staticJavaFxLibs = Arrays.asList(
             "glass", "javafx_font", "javafx_iio", "prism_es2"
     );
+    private static final List<String> macfxWebkit = List.of("jfxwebkit");
+    private static final List<String> macfxWeblibs = List.of(
+            "-lWebCore", "-lXMLJava", "-lJavaScriptCore", "-lbmalloc",
```

Comment:
For consistency, use a List without specifying `-l` for each entry, ie: `"WebCore", "XMLJava", "JavaScriptCore", ...`. Then add this list applying the `asListOfLibraryLinkFlags(libs)` construct.

Reply:
makes sense, done